docs/howtos/policy-servers/03-production-deployments.md

```diff
@@ -0,0 +1,209 @@
+they process Admission Requests destined for the Kube API via the Validating and
+Mutating Webhooks.
+
+As with other Dynamic Admission Controllers, this process happens before
+requests reach the Kube API server. Latency or service delays by
+the Dynamic Admission Controller may introduce cluster inconsistency,
+Denial of Service, or deadlock.
+
+Kubewarden provides several ways to increase the reliability of PolicyServers.
+Production deployments can vary a great deal, it is up to the operator to configure the deployment for their needs.
+
+# PodDistruptionBudgets
+
+The Kubewarden controller can create a
+[PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)
+(PDB) for the policy-server Pods. This controls the range of policy-server
+Pod replicas associated with the PolicyServer, ensuring high availability
+and controlled eviction in case of node maintenance, scaling operations or
+cluster upgrades.
+
+This is achieved by setting `spec.minAvailable`, or `spec.maxUnavailable` of the
+PolicyServer resource:
+
+- `minAvailable`: specifies the minimum number of policy-server Pods
+  that must be available at all times. Can be an integer or a percentage.
+
+  Useful for maintaining the operational integrity of the PolicyServer,
+  ensuring that policies are continously enforced without interruption.
+
+- `maxUnavailable`: specifies the maximum number of policy-server Pods that can
+  be unavailable at any given time. Can be an integer or a percentage.
+
+  Useful for performing rolling updates or partial maintenance without fully
+  halting the policy enforcement mechanism.
+
+:::note
+You can specify only one of `maxUnavailable` and `minAvailable`.
+:::
+
+## Configuring minAvailable or maxUnavailable
+
+Examples:
+
+```yaml
+apiVersion: policies.kubewarden.io/v1
+kind: PolicyServer
+metadata:
+  name: your-policy-server
+spec:
+  # Other configuration fields
+  minAvailable: 2 # ensure at least two policy-server Pods are available at all times
+```
+
+```yaml
+apiVersion: policies.kubewarden.io/v1
+kind: PolicyServer
+metadata:
+  name: your-policy-server
+spec:
+  # Other configuration fields
+  maxUnavailable: "30%" # ensure no more than 30% of policy-server Pods are unavailable at all times
+```
+
+# Affinity / Anti-affinity
+
+The Kubewarden controller can set the affinity of policy-server Pods. This
+allows constraint of Pods to specific nodes, or Pods against other Pods. For
+more information on Affinity, see the [Kubernetes
+docs](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity).
+
+Kubernetes affinity configuration allows constraining Pods to nodes (via
+`spec.affinity.nodeAffinity`) or constraining Pods with regards to other Pods
+(via `spec.affinity.podAffinity`). Affinity can be set as a soft constraint
+(with `preferredDuringSchedulingIgnoredDuringExecution`) or a hard one (wit
+`requiredDuringSchedulingIgnoredDuringExecution`). Via labels.
```

Comment:
I'm not sure the 'Via labels' at the end is necessary, and I think if it is, does it need to be with the next paragraph?